modules/installation-restricted-network-samples.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-restricted-network-samples_{context}"]
+= Using sample imagestreams in a restricted network installation
+
+Most imagestreams in the OpenShift namespace managed by the Samples Operator
+point to images located in the Red Hat registry at registry.redhat.io. Mirroring
+will not apply to these imagestreams.
+
+The `jenkins`, `jenkins-agent-maven`, and `jenkins-agent-nodejs` imagestreams do
+come from the install payload and are managed by the Samples Operator, so no
+further mirroring procedures are needed for those imagestreams.
+
+[NOTE]
+====
+The `cli`, `installer`, `must-gather`, and `tests` imagestreams, while
+part of the install payload, are not managed by the Samples Operator. These are
+not addressed in this procedure. 
+====
+
+.Prerequisites
+* Cluster administrator privileges
+
+.Procedure
+
+. Mirror images from registry.redhat.io associated with any imagestreams you need
+in the restricted network environment into one of the defined CRI-O mirrors.
```

Comment:
I'm not actually sure this is documented anywhere. I will need to track it down, and if not we will need to add the procedure.

Reply:
@gabemontero is the CRI-O mirror concept the same as creating a mirror registry?

https://docs.openshift.com/container-platform/4.2/installing/installing_restricted_networks/installing-restricted-networks-preparations.html#installation-creating-mirror-registry_installing-restricted-networks-preparations

Reply:
certainly move off of `CRI-O mirrors` to whatever is the more acceptable terminology as you see fit